src/components/StayInformed/SignUp/styles.js

```diff
@@ -0,0 +1,36 @@
+export default theme => ({
+  container: {
+    width: '631px',
+    height: '286px',
+    backgroundColor: '#e9e9e9',
```

Comment:
we should import these colors from the global theme and if theyre not there we should add them in